cypress/e2e/5SummaryFeature.cy.js

```diff
@@ -12,15 +12,14 @@ describe("Home", () => {
 
     it("Check for correct word length", () => {
         cy.get("#inputted-text").type(
-            "Machine learning (ML) is an umbrella term for solving problems for which development of algorithms by human programmers would be cost-prohibitive, and instead the problems are solved by helping machines discover their own algorithms, without needing to be explicitly told what to do by any human-developed algorithms. Recently, generative artificial neural networks have been able to surpass results of many previous approaches. Machine-learning approaches have been applied to large language models, computer vision, speech recognition, email filtering, agriculture and medicine, where it is too costly to develop algorithms to perform the needed tasks. The mathematical foundations of ML are provided by mathematical optimisation (mathematical programming) methods. Data mining is a related (parallel) field of study, focusing on exploratory data analysis through unsupervised learning."
-        );
+            "Machine learning (ML) is an umbrella term for solving problems for which development of algorithms by human programmers would be cost-prohibitive, and instead the problems are solved by helping machines discover their own algorithms, without needing to be explicitly told what to do by any human-developed algorithms. Recently, generative artificial neural networks have been able to surpass results of many previous approaches. Machine-learning approaches have been applied to large language models, computer vision, speech recognition, email filtering, agriculture and medicine, where it is too costly to develop algorithms to perform the needed tasks. The mathematical foundations of ML are provided by mathematical optimisation (mathematical programming) methods. Data mining is a related (parallel) field of study, focusing on exploratory data analysis through unsupervised learning.",
+            { delay: 0 }
+          );
 
         cy.get("#summarise-button-id").click();
 
-        cy.wait(12000);
-
         // Assert that the text contains the sentiment and score
-        cy.get("#summary-result").then(($div) => {
+        cy.get("#summary-result", {timeout: 20000}).then(($div) => {
```

Comment:
btw `.then()` will not automatically retry the command until the assertion passes. `.should()` is more robust and will retry the assertion until it passes or the timeout finishes

Reply:
Good pickup, have updated this test to now use `.should()`